### PR TITLE
A manager should assign users to managers and be also able to view a list of verified users with their managers

### PR DESCRIPTION
**Pull request by freddykaberuka**
## What does this PR do?
Assign User to a Manager

## Description of Task to be completed

As **a manager** I want to be able to assign **verified user** to their manager, so that the user can start making travel requests.
## [the tasks that you completed on the feature]

1. The **manager** should be able to assign a verified user to a manager
2. Appropriate error messages are displayed on unsuccessful attempts.
3. Appropriate confirmation messages are displayed on successful requests.
4. 

## How should this be manually tested?
Open your terminal and navigate to the appropriate directory where you want to store your project.
git clone ....
git checkout ft-assignUserstoManagers
npm install
add environment variable to your .env
npm start

## What are the relevant Trello story?
https://trello.com/c/m5kcAW5G/43-assign-users-to-managers

### for requesting assigning a manager
Screen Shot 2021-02-10 at 13 39 06

### CODE REVIEW

File: src/services/userService.js [hunk 1]
```diff
@@ -62,5 +62,16 @@ class UserService {
...
+    return Users.findOne({where:
+    {id: lineManagerId},
+  });
+
+  }
+  // static findAll(data){
+  //   return Users.findAll({
+  //     where:{isVerified=='true' && lineManager}
+  //   })
+  // }
```
Comment (bertinNDAYISHIMIYE) [inline src/services/userService.js]: remove unnecessary codes!

**Review by bertinNDAYISHIMIYE — DISMISSED** [review]
LGTM, but left you a comment above! and protect your routes unless any user can assign a manager.

**Review by bertinNDAYISHIMIYE — COMMENTED** [review]
Your branch has some conflicting files.

**Review by nkalyesubula — APPROVED** [review]
LGTM

**Pull request merged**